src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -12,40 +12,44 @@ redirects:
+    ![](/src/images/synthetics-create-monitor.png)
+2. Select a monitor type and then fill in all required fields.
+3. You have the option to add tags, change the **Period** or [Runtime version](/docs/synthetics/synthetic-monitoring/using-monitors/new-runtime)(available for simple browser, scripted browser and API monitors), or add a validation string (available for ping and simple browser monitors), or use advanced options, which enable substring monitoring for the following types of response validation:
 
    * **Verify SSL (for ping and simple browser).** This option verifies the validity of the SSL certificate chain. It can be duplicated by running the following syntax:
 
-     ```
+     ```sh
      openssl s_client -servername <var>{YOUR_HOSTNAME}</var> -connect <var>{YOUR_HOSTNAME</var>}:443 -CApath /etc/ssl/certs > /dev/null
      ```
-   * **Bypass HEAD request (for ping).** This option skips the default HEAD request and instead uses the GET verb with a ping check.
-   * **Redirect is Failure (for ping).** If a redirect result occurs when Redirect is Failure is enabled, Synthetics categorizes it as a failure (rather than following the redirect and checking the resulting URL).
-3. Select the locations where you want your monitor to run.
-4. Choose a frequency to determine how often each location will run your monitor.
-5. Optional: Set up alert notifications.
-6. Select **Create my monitor** to confirm.
-7. Wait a few minutes; then from the [Monitors index](#bookmark) check your monitor.
 
-## 2. View all monitors with the Monitors index page [#bookmark]
+   * **Bypass HEAD request (for ping monitors).** This option skips the default HEAD request and instead uses the GET verb with a ping check. GET requests will always happen if the HEAD request fails.
+   * **Redirect is Failure (for ping).** If a redirect result occurs when `Redirect is Failure` is enabled, your synthetic monitors categorize it as a failure (rather than following the redirect and checking the resulting URL).
+
+4. Select the locations where you want your monitor to run. It is recommended to select at least three locations to avoid false positives. In other words, if at least one location returns a successful result then the endpoint must be up and triggering of an alert can be avoided.
+5. Depending on the monitor type, choose to either Save monitor, Validate, or Write script.
+6. You'll be taken to the [Summary page](#summary-page) where results will start to appear as they are received.
+
+## 2. Get a quick summary of how a monitor is running [#summary-page]
+
+TBD, screenshots, what's useful on this page?
+
+Once on the Summary page, you'll be presented with the most relevant information to assist you in determining the status of this monitor. If an alert has triggered and created an active incident, click to open the critical alert in a new tab for this monitor, or click to manage alert policies for all monitors at the top of the page.
```

Comment:
@keegoid-nr What precisely is opening in a new tab? Is it a summary page of the incident itself?

Reply:
What is being clicked and how does it appear in the UI? Do we have screenshots of this?

Reply:
@akristen I did not get this part done yet.